deepspeed/runtime/bf16_optimizer.py

```diff
@@ -75,7 +75,6 @@ def __init__(self,
         self.fp32_groups_gradient_flat_partition = []
         self.fp32_groups_has_gradients = []
 
-        self.step_count = 0
```

Comment:
just checking - you removed this counter because it actually wasn't used anywhere?

Reply:
correct